frank-doc-frontend/src/app/components/collapse.directive.spec.ts

```diff
@@ -5,8 +5,9 @@ import { By } from '@angular/platform-browser';
 
 @Component({
   template: ``,
-  imports: [CollapseDirective],
-  standalone: true,
+  imports: [
+    /*CollapseDirective*/
+  ],
```

Comment:
Is this intended?

Reply:
Its used if we actually wrote karma tests, but since they are now blank we don't really do anything with the import